Slim/Utils/Log.pm

```diff
@@ -820,6 +820,9 @@ sub logGroups {
 				'database.info'          => 'DEBUG',
 				'plugin.itunes'          => 'DEBUG',
 				'plugin.musicip'         => 'DEBUG',
+				'database.virtuallibraries' => 'DEBUG'
+				'formats.audio'          => 'ERROR',
```

Comment:
Why would you add these two? `ERROR` is what they're set to by default, isn't it?

Reply:
I was proposing to add them because if they are not added, the scanner (when launched by `Slim::Music::Import::launchScan`) will simply not see the current (running) log levels, but will only respond to the persistent log level in `log.conf`. So what you see is not what you get.

I think that my real "complaint" is this:

The scanner **should** see **all** the current (running) log levels when it is launched from LMS (either through the UI or a scheduled run). That is what I would expect if I didn't know any better.

I think this could be easily achieved with a small modification to `Slim::Music::Import::launchScan`, whereby it launches the scanner process with a clone of the *current* log settings.

From my (restricted) point of view, that would be a better approach. I will draft something and post it here.

Reply:
I have submitted a new pull request, https://github.com/Logitech/slimserver/pull/253, which, I think, deals with the point in a better way.

Reply:
If that's accepted I would suggest this PR simply be deleted.

Reply:
My question was more about the log level of `ERROR` you chose, when that's the default anyway, and that group should increase logging for the given group of categories.